readme-examples/shared/src/main/scala/kyo/utils.scala

```diff
@@ -0,0 +1,43 @@
+package kyo.readmeExamples
+
+import scala.io.Source
+
+object utils:
```

Comment:
@mobley-trent apologies for the delay to review! Can you clarify how the solution works? Do we need to use this implementation manually to format the readme code?

Reply:
@fwbrasil this was actually a solution proposed by @987Nabil but I went on to use your original approach of putting the code inside scala files and running scalafmt. I actually meant to remove the rest of the redundant files